cw/capture.py

```diff
@@ -60,6 +63,16 @@ def create_cw_segmented(ot, capture_cfg):
 }
 
 
+def abort_handler(project, sig, frame):
+    """ Handler for ctrl-c keyboard interrupts:
+        Saves capture project before exiting, in case abort is intended.
+        Needs to be registered in every capture function before capture loop. """
+    if project is not None:
+        print("caught keyboard interrupt -> saving project (traces)...")
+        project.save()
+    sys.exit(0)
```

Comment:
This has to go into a separate PR, because it's unrelated to OTBN vertical analysis. Instead it's useful for any captures and a great addition actually, thanks for adding this!

Reply:
Thanks @m-temp and @wettermo!

Reply:
I will remove that from the current PR